src/wpt/main.js

```diff
@@ -20,27 +20,57 @@ if (test) {
 
 	const { requests, from, performance, testUrl } = await get_result(test);
 
+	/** requests below this size will be grouped */
+	const threshold = requests.reduce((total, { objectSize }) =>
+		total + objectSize, 0) / 360;
+
+	/** requests below this size will be discarded */
+	const discard = 100;
+
+	const { above, below, discarded } = requests.reduce(
+		(accumulator, request) => {
+			if (request.objectSize > threshold) accumulator.above.push(request);
+			else if (request.objectSize > discard) accumulator.below.push(request);
+			else accumulator.discarded.push(request);
+			return accumulator;
+		},
+		{
+			above: /** @type {Requests} */ ([]),
+			below: /** @type {Requests} */ ([]),
+			discarded: /** @type {Requests} */ ([]),
+		},
+	);
+
+	if (discarded.length > 1) {
+		console.warn(
+			`The following ${discarded.length} requests were discarded:`,
+			discarded,
+		);
+	}
```

Comment:
I think this is fine, but it does mean that we don't have much visibility on teeny requests